docs/tutorial/chat2.md

```diff
@@ -62,6 +83,19 @@ In order to connect to a *specific* node as [`relay`](https://specs.vac.dev/spec
 
 This will bypass the random peer selection process and connect to the specified node.
 
+## Connecting to a Waku v2 fleet
+
+It is possible to specify a specific Waku v2 fleet to connect to when starting the app by using the `--fleet` option:
+
+```
+./build/chat2 --fleet:test
+```
+
+There are currently two fleets to select from, namely `prod` and `test`.
+The `test` fleet is updated more regularly so may have more advanced features, but will be less stable as a result.
+If no `fleet` is specified, `chat2` will connect to the `prod` fleet by default.
+To start `chat2` without connecting to a fleet, use the `--fleet:none` option _or_ [specify a static peer](#specifying-a-static-peer).
```

Comment:
I'd explain what is the benefit of each type of `fleet`, also what is the use of being connected to fleet node? to fetch historical messages?

Reply:
Done in f8505c8